packages/flutter/lib/src/cassowary/solver.dart

```diff
@@ -275,6 +291,7 @@ class Solver {
     return _bulkEdit(variables, applier, undoer);
   }
 
+  /// Attempt to remove the specified edit variable from the solver.
```

Comment:
Attempts.

General comment for a lot of these: I think we generally don't both saying "Attempts to...", we just say it does it, and if it might fail, we then have a discussion of why it might fail and how that will manifest.